.travis.yml

```diff
@@ -0,0 +1,126 @@
+# 'sudo required' will give us 7,5 GB memory, but has slower startup times, as we start a VM instead of a Container
+sudo: required
+language: python
+cache: pip
+
+python: 2.7
+
+jobs:
+  include:
+    - stage: lint
+      addons: {}
+      before_install: skip
+      install:
+        - pip install flake8 flake8-import-order planemo
+        - planemo --version
+        - git diff --quiet "$TRAVIS_COMMIT_RANGE" -- ; GIT_DIFF_EXIT_CODE=$?
+        - |
+          if [ "$GIT_DIFF_EXIT_CODE" -gt 1 ] ; then
+              git remote set-branches --add origin master
+              git fetch
+              TRAVIS_COMMIT_RANGE=origin/master...
+          fi
+        - echo $TRAVIS_COMMIT_RANGE
+        - |
+          planemo ci_find_repos --exclude_from .tt_blacklist \
+                                --changed_in_commit_range "$TRAVIS_COMMIT_RANGE" \
+                                --output changed_repositories.list
+        - cat changed_repositories.list
+      script:
+        - set -e
+        - cd "$TRAVIS_BUILD_DIR" && flake8 --exclude=.git,./deprecated/ tools/
+        - while read -r DIR; do planemo shed_lint --tools --ensure_metadata --urls --report_level warn --fail_level error --recursive "$DIR"; done < changed_repositories.list
+
+    - stage: test
+      env: CHUNK=0
+
+    - stage: test
+      env: CHUNK=1
+
+    - stage: test
+      env: CHUNK=2
+
+    - stage: test
+      env: CHUNK=3
+
+    - stage: deploy
```

Comment:
Will this skip over uploading the tool suite?

Reply:
Changed .travis to only look in snvphy-tools directory so it will never auto deploy tool suite.